ONSdigital/design-system
# Bugfix/1581/input description voiceover mac

### What is the context of this PR?
The aria-labelled by label on a prefixed or suffixed input was preventing both the units and label being read out on Mac Voiceover.

https://github.com/ONSdigital/design-system/issues/1581

### How to review
1. Load Netlify preview of the component on the PR.
2. Check component behaviour for `example-input-number-prefixed` &  `example-input-number-suffixed` on Mac Voiceover. Both the label and the unit should be read out to the user on focus of the input.
